### PR TITLE
Implement FIXED, TEXT & DOLLAR functions

### PR DESCRIPTION
**Pull request by andraz**

### CODE REVIEW

**Comment by nicolashefti** [comment]
Hi @andraz , thanks for the contribution.

Formulajs is a fully unit tested library. You need to provide test cases associated with the new function. You can have a look at NUMBERVALUE or VALUE for some inspiration.

File: src/text.js [hunk 1]
```diff
@@ -224,10 +221,34 @@ export function FIND(find_text, within_text, start_num) {
  * @param {*} no_commas Optional. A logical value that, if TRUE, prevents FIXED from including commas in the returned text.
  * @returns
  */
-export function FIXED() {
-  throw new Error('FIXED is not implemented')
+export function FIXED(number_, decimals_, no_commas) {
+  const number = utils.parseNumber(number_)
+  if (isNaN(number)) {
+    return '#VALUE!'
```
Comment (nicolashefti) [inline src/text.js:227]: There are already existing error Object (see ./utils/errors.js)

File: src/text.js [hunk 2]
```diff
@@ -224,10 +221,34 @@ export function FIND(find_text, within_text, start_num) {
  * @param {*} no_commas Optional. A logical value that, if TRUE, prevents FIXED from including commas in the returned text.
  * @returns
  */
-export function FIXED() {
-  throw new Error('FIXED is not implemented')
+export function FIXED(number_, decimals_, no_commas) {
```
Comment (nicolashefti) [inline src/text.js:224]: Why _number__ and not just _number_

Comment (andraz) [reply src/text.js:224]: Overwriting the incoming parameters is a bad code practice. They should be considered as `const`.

Comment (nicolashefti) [reply src/text.js:224]: But in this case you should keep the clearer name (= "number") for outside the function and use another the one for the implementation.

**Comment by andraz** [comment]
TEXT partially implemented, as there is another PR opened on the topic further development can grow from it

**Comment by nicolashefti** [comment]
@andraz it looks good to me. The TEXT implementation looks pretty neat to me. Indeed we can grow from it.

Is it ready for you? Ready to merge when you are.

**Comment by andraz** [comment]
Yes, all tests are passing so it is good to go.

**Review by nicolashefti — APPROVED** [review]

**Pull request merged**